exercises/csharp-1-a/Csharp1A.cs

```diff
@@ -1,7 +1,9 @@
+using System;
```

Comment:
This helps people just do `Console.WriteLine` instead of `System.Console.WriteLine`.